protocol/contracts/interfaces/amo/helpers/IRamosTokenVault.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (interfaces/amo/helpers/ITokenVault.sol)
```

Comment:
Update the path here please

(NB I use this such that when someone looks at the verified contract in etherscan, they know where it is in gh)